packages/frontend/src/features/catalog/components/CatalogTree.tsx

```diff
@@ -127,6 +134,9 @@ export const CatalogTree: FC<React.PropsWithChildren<Props>> = ({
         return null;
     }
 
+    const hideGroupedTables =
+        Object.keys(tree).length === 1 && tree['Ungrouped tables'];
```

Comment:
sorry I don't get this. Can you explain?

Reply:
If there is only one group and it is the ungrouped tables group, we don't show the group name (which would be 'Ungrouped tables'). Hiding this wasn't in the designs but it felt weird to have one unnamed node there.